### DISCUSSION TITLE
Alist识别不到NAS的外部挂载硬盘，NAS使用的是玩客云，外部挂载是125GB的内存卡

### DISCUSSION THREAD

**Discussion by Sika-Liu** [AlistGo/alist#7289] [Q&A]
在1panel面板中是可以找到外部挂载的路径及文件夹，Alist中识别不到，并且Alist中的media里面的文件和面板中的media中的文件不一样，  版本是v3.37.4

![1](https://github.com/user-attachments/assets/c77f1f49-8e9a-4fd5-a64d-15eccae19432)
![2](https://github.com/user-attachments/assets/119646df-f418-44af-890f-79cedc60b996)
![3](https://github.com/user-attachments/assets/2568ba3a-31b6-4a28-a3d3-42619f0a02c0)
![4](https://github.com/user-attachments/assets/c0f4696c-4657-4c3b-9231-9bee013c6494)
![5](https://github.com/user-attachments/assets/5945cd31-d168-4a20-9dee-77d9597cf0ca)
![6](https://github.com/user-attachments/assets/c6df31c1-f5b7-417f-a7a6-154d005239be)
![7](https://github.com/user-attachments/assets/a64a60f6-f48e-4035-8a4c-11b1e7ed25ad)

**Comment by pongfcnkl** [comment]
alist是root启动的吗
如果是docker搭建的，需要映射

**Comment by Simonzhang8** [comment]
看样子你是docker运行的alist，需要配置docker映射目录
